packages/react-theme-provider/src/compat/makeClasses.ts

```diff
@@ -1,5 +1,5 @@
 import { IStyle } from '@fluentui/merge-styles';
-import { Theme } from '@fluentui/theme';
+import { Theme } from './types';
```

Comment:
Does this mean that we can drop `@fluentui/theme` in `package.json` of `react-theme-provider`?

Reply:
not yet, still need it for accessing the default theme